xds/src/main/java/io/grpc/xds/LookasideLb.java

```diff
@@ -147,17 +150,26 @@ public int getPriority() {
         }
 
         /**
-         * A synthetic policy name identified by balancerName. The implementation detail doesn't
+         * A synthetic policy name identified by xds config. The implementation detail doesn't
```

Comment:
This should be an explanation for your implementation. Head comment should only talk about representative specification. Also, this is an override method, no head comment is necessary, parent class should have this explained.

Reply:
Used `//` and removed "The implementation detail doesn't matter".